sublime-package.json

```diff
@@ -739,6 +739,12 @@
               "default": false,
               "markdownDescription": "Show inlay hints in the editor. Inlay hints are short annotations within the code, which show variable types or parameter names.\nThis is the default value used for new windows but can be overriden per-window using the `LSP: Toggle Inlay Hints` command from the Command Palette, Main Menu or a custom keybinding."
             },
+            "inlay_hints_max_length": {
+              "type": "integer",
+              "default": 100,
+              "minimum": 0,
+              "markdownDescription": "Truncate limit for inlay hints. Truncates and adds an ellipsis at the end."
```

Comment:
Update to match description in settings?

Reply:
Done in [dc7d5ca](https://github.com/sublimelsp/LSP/pull/2514/commits/dc7d5ca31f0918a586659d12ee735ca17c18a215)